dev-docs/bidders/lucead.md

```diff
@@ -27,6 +27,26 @@ The Lucead Bidding adapter requires setup before beginning. Please contact us at
 
 ### Bid Params
 
+{: .table .table-bordered .table-striped }
 | Name          | Scope    | Description           | Example   | Type      |
 |---------------|----------|-----------------------|-----------|-----------|
 | `placementId` | required | Placement id          | `'11111'` | `string`  |
+
+### Test Parameters
+
+```javascript
+const adUnits = [
+       {
+           code: 'test-div',
+           sizes: [[300, 250]],
+           bids: [
+               {
+                   bidder: 'lucead',
+                   params: {
+                       placementId: '1',
```

Comment:
Not sure if it's significant, but flagging in any case. Your accompanying Prebid.js PR changes this value  to `2`.

Reply:
Thanks for the feedback. `1` is a correct value for testing.